### ISSUE TITLE
Range usage

### ISSUE THREAD

**Issue by jonathanolson** [phetsims/john-travoltage#159]
There are a number of ad-hoc usage of Range-like objects with min/max defined, e.g. AppendageRangeMaps:
```js
range: {
    max: 5,
    min: 0
},
```

The Range object in dot should be preferred in those cases.

Code review for https://github.com/phetsims/john-travoltage/issues/128

**Comment by jessegreenberg** [comment]
Actually, this is related to a11y, I will work on this.

**Comment by jessegreenberg** [comment]
Also the strings should be in JohnTravoltageA11yStrings.

**Comment by jessegreenberg** [comment]
Done in the above commits, closing.
